app/views/participants/fields/_family_history.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= f.inputs :name => "Family History Information" do -%>
 
-    <%= f.input :first_generation, :as => :select, :label => "Did parents graduate from college?", :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } %>
+    <%= f.input :first_generation, :as => :select, :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } unless @current_user.person_type == "Participant" %>
+
+    <%= f.input :first_generation, :as => :select, :collection => { "No (I am first generation to college)" => true, "Yes (I am NOT first generation to college)" => false } unless @current_user.person_type != "Participant" %>
```

Comment:
No I don't think so. But I'd recommend defining the collection before this line, using an `if ... else` and then specifying that in the `f.input`. You should avoiding repeating all of the other parts of the line.